src/components/Dashboard.tsx

```diff
@@ -91,12 +92,10 @@ const TemplateComponent: React.FC<{ template: TemplateInList }> = ({ template })
   )
 }
 
-const FilterComponent: React.FC<{ template: TemplateDetails; filter: Filter; setLoading:(loading:boolean) => void; filters: Array<{}>; i:number }> = ({
+const FilterComponent: React.FC<{ template: TemplateDetails; filter: Filter; setLoadingFilters:Function; filters: Array<{}>; loadingFilters: number}> = ({
```

Comment:
You don't need to pass `loadingFilters` down to the FilterComponent. That component doesn't need to know the overall state of the filters, it just needs a mechanism to update it, which it does through `setLoadingFilters`.

Reply:
Type for "setLoadingFilters" should be something like `setLoadingFilters: (setState: Function) => void`
And you can remove the "filters" as mentioned earlier.